src/schema/v2/partner/partner.ts

```diff
@@ -93,30 +93,34 @@ const artworksArgs: GraphQLFieldConfigArgumentMap = {
   forSale: {
     type: GraphQLBoolean,
   },
+  includeUnpublished: {
+    type: GraphQLBoolean,
+    description:
+      "If true return both published and unpublished artworks, requires auth",
+  },
   missingPriorityMetadata: {
     type: GraphQLBoolean,
     description: "Return artworks that are missing priority metadata",
   },
+  offerable: {
```

Comment:
#non-blocking. I have a similar comment about using the explicit name: `partner_offerable` to avoid confusing ourselves with MO. See if that makes sense to you.